packages/template-retail-react-app/app/components/header/index.test.js

```diff
@@ -166,18 +134,10 @@ test('renders cart badge when basket is loaded', async () => {
 })
 
 test('route to account page when an authenticated users click on account icon', async () => {
-    global.server.use(
-        rest.post('*/customers/action/login', (req, res, ctx) => {
-            return res(ctx.delay(0), ctx.status(200), ctx.json(mockedRegisteredCustomer))
-        })
-    )
-    useCustomerType.mockReturnValue({isRegistered: true, customerType: 'registered'})
     const history = createMemoryHistory()
     // mock push function
     history.push = jest.fn()
-    await act(async () => {
-        renderWithProviders(<MockedComponent history={history} />)
-    })
+    renderWithProviders(<MockedComponent history={history} />)
```

Comment:
Yess, we simply need to call renderWithProviders 👍 because `act` is already called within render. And no need to `await` render since render is not an asynchronous function and it returns an object.